fab/pillows/production.yml

```diff
@@ -5,27 +5,21 @@ exclude_groups:
 # include_groups:
 #   - mvp_indicators
 include_pillows:
-#   - custom.bihar.models.CareBiharFluffPillow
   - custom.opm.models.OpmUserFluffPillow
   - custom.reports.mc.models.MalariaConsortiumFluffPillow
-#   - custom.m4change.models.AncHmisCaseFluffPillow
-#   - custom.m4change.models.LdHmisCaseFluffPillow
-#   - custom.m4change.models.ImmunizationHmisCaseFluffPillow
-#   - custom.m4change.models.ProjectIndicatorsCaseFluffPillow
-#   - custom.m4change.models.McctMonthlyAggregateFormFluffPillow
-#   - custom.m4change.models.AllHmisCaseFluffPillow
+  - custom.m4change.models.M4ChangeFormFluffPillow
```

Comment:
The checkpoint for this should be set to 49852429 after deploy (minimum of the other checkpoints) I'll do this